src/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from './types';
+
```

Comment:
Before starting to work with functions and components, it would be very cool to define the variables that we use further into separate constants, so that we do not use magic values:
```
const FATHER = 'father';
const MOTHER = 'mother';
const FEMALE = 'f';
const NOT_SET_VALUE = '-';
```
This will provide us with better readability of the code itself, conditions in the code and understanding that we will not accidentally make a mistake in a line.